library/src/main/java/in/krharsh17/barview/BarView.java

```diff
@@ -161,12 +175,23 @@ else if (animationType == BarView.INTRO_ANIM_EXPAND){
         }
     }
 
+    /**
+     * This method iterates over various BarModels and adds a BarGroup for each
+     * of the BarModels inside private member data
+     */
     private void populateBarView(int animationType, int animationDuration) {
         for (BarModel b : data) {
             addBar(b,animationType,animationDuration);
         }
     }
 
+    /**
+     * This method actuall instanciates BarGroups based on the BarModel passed in as
+     * a param and cumulates it into barGroups.
+     * @param data is a BarModel that contains all the required to
+     *             construct a BarGroup instance.
+     */
+
```

Comment:
```suggestion
    /**
     * This method actually instantiates BarGroups based on the BarModel passed in as
     * a param and cumulates it into @param barGroups.
     * @param data is a BarModel that contains all the data required to
     *             construct a BarGroup instance.
     */

```

Reply:
@krharsh17 dont we use the @param notation only for the params passed to the function.

Reply:
I guess we use @link for it ? Not sure btw.

Reply:
Yeah, I think you're right. Just confirm it once though.